src/inmanta/app.py

```diff
@@ -97,6 +106,9 @@ def start_server(options: argparse.Namespace) -> None:
     if options.db_wait_time is not None:
         Config.set("database", "wait_time", str(options.db_wait_time))
 
+    if options.compatibility_file:
+        Config.set("server", "compatibility_file", str(options.compatibility_file))
```

Comment:
I don't think this is going to work. The option has a default value so the if-statement will always validate to True. Like that we will always ignore the config value set in the configuration file. I think we have to set the default value to None so that we can determine whether the option is set on the comandline or not.